File: app/src/main/java/ww/futsimul/MainActivity.java
package ww.futsimul;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import ww.futsimul.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle saveInstanceState) {
        super.onCreate(saveInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/ww/futsimul/MainActivity.kt
package ww.futsimul

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import ww.futsimul.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {


    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

    //setContentView(R.layout.activity_main)
       //val tvHello = findViewById<TextView>(R.id.tvHello)

       binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.tvHello.text = "É gooooolll"
    }
}